app/styles/prototype/_search.scss

```diff
@@ -0,0 +1,77 @@
+.auk-card-table {
```

Comment:
Are these changes just a copy/paste from the `usertest` branch? If so, I'll give them a more thorough look later on (probably next sprint).

Reply:
Yes, exactly! Thanks.